### ISSUE TITLE
Cannot read property 'CognitoUserAttribute' of undefined

### ISSUE THREAD

**Issue by GuzmanPI** [awslabs/aws-cognito-angular2-quickstart#64]
Hi guys. This is probably not an issue with this repo, but I wanted to contact you.

I have an Angular CLI app and I'm adding Cognito signup/login based on this project. However, I keep getting this error:

TypeError: Cannot read property 'CognitoUserAttribute' of undefined at UserRegistrationService

I do not know what I'm missing!

I have all the .js in assets/js folder

And I have added al the scripts in the index.

Thanks in advance for the help

**Comment by vbudilov** [comment]
Have you made any modifications to the original code?

**Comment by GuzmanPI** [comment]
I had an Angular CLI app already, and I'm trying to integrate Cognito. 

I copied the JS folder into the assets folder, put the scripts in the index.html, and I copied the cognito.service.

I implemented the interface, and call the signup method from my component, but when it gets to de service It gives that error.

I think is something to do with the aws' sdk not being imported or something like that...

Thanks for your time!
